exchange/exchange_test.go

```diff
@@ -2290,14 +2290,49 @@ func TestTimeoutComputation(t *testing.T) {
 	ctx, cancel := context.WithDeadline(context.Background(), deadline)
 	defer cancel()
 
-	auctionCtx, cancel := ex.makeAuctionContext(ctx, true)
+	auctionCtx, cancel := ex.makeAuctionContext(ctx, true, &config.TmaxAdjustments{})
 	defer cancel()
 
 	if finalDeadline, ok := auctionCtx.Deadline(); !ok || deadline.Add(-time.Duration(cacheTimeMillis)*time.Millisecond) != finalDeadline {
 		t.Errorf("The auction should allocate cacheTime amount of time from the whole request timeout.")
 	}
 }
 
+func TestMakeAuctionContextWithTmaxAdjustment(t *testing.T) {
+	upstreamResponseDuration := 30000 //in milliseconds
```

Comment:
Generally `Tmax` set is near to 500-700ms. How about using 100ms as `upstreamResponseDuration`. This is more near to actual vale

Reply:
This is kinda subjective relative to machine that we are using. For example - this value might be different for different machine based on various factors like CPU utilisation, operating system, number of cores, number of processes running etc. But ok, will change it.

Reply:
https://github.com/prebid/prebid-server/pull/2750/commits/3c5167ea79f099afb0eeaf1f5398f4b444c4135c